src/Config.js

```diff
@@ -40,6 +40,16 @@ class Config {
     return process.env.SINOPIA_URI || 'https://sinopia.io'
   }
 
+  static get sinopiaEnv() {
+    if (process.env.SINOPIA_ENV) {
+      return `- ${process.env.SINOPIA_ENV}`
+    }
+    else
+    {
+      return ''
+    }
```

Comment:
Purely aesthetic, but I have a preference for the more guard-clause-y incantation if you are open to it:

```javascript
if (process.env.SINOPIA_ENV) {
  return `- ${process.env.SINOPIA_ENV}`
}

// We do not set this value in production, and don't want to see the env label in production
return ''
```